### PR TITLE
Add cpu + memory limits

### PR DESCRIPTION
**Pull request by Zohar2506**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by cpanato** [comment]
@Zohar2506 Please sign the DCO and bump the chart version

File: stable/rookout/templates/deployment.yaml [hunk 1]
```diff
@@ -23,6 +23,10 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          resources:
+          limits:
+            memory: "0.5Gi"
```
Comment (cpanato) [inline stable/rookout/templates/deployment.yaml]: I would say these as variables

Comment (Zohar2506) [reply stable/rookout/templates/deployment.yaml]: Done 👍

Comment (Zohar2506) [reply stable/rookout/templates/deployment.yaml]: @cpanato what is a DCO and what do you mean by removing the chart version?

File: stable/rookout/Chart.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: "1.0"
 description: A Helm chart for Rookout agent on Kubernetes
 icon: https://app.rookout.com/static/rookout/img/logo.svg
 name: rookout
 version: 0.1.0
 home: "http://rookout.com/"
 maintainers:
-- name: Zohar
+- name: Zohar Einy
```
Comment (cpanato) [inline stable/rookout/Chart.yaml]: should be the github username

File: stable/rookout/values.yaml [hunk 3]
```diff
@@ -12,6 +12,10 @@ tags: ''
 rookout:
   token: ''
 
+limits:
+  memory: 0.5Gi
+  cpu: 0.5
```
Comment (cpanato) [inline stable/rookout/values.yaml]: please add the scale similar what you did for memory

**Comment by Zohar2506** [comment]
Can you take a look at the DCO error? for some reason it does not approve to test although I added the signed of by

**Comment by cpanato** [comment]
still missing the DCO -  https://www.helm.sh/blog/helm-dco/index.html

**Comment by Zohar2506** [comment]
It says I have one commit behind - https://github.com/helm/charts/pull/7867/checks?check_run_id=19396273

**Comment by cpanato** [comment]
@Zohar2506 you can rebase and squash the commits and then sign off

**Comment by Zohar2506** [comment]
@cpanato I added the appVersion but it fails Circle ci

File: stable/rookout/Chart.yaml [hunk 4]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: "1.0"
 description: A Helm chart for Rookout agent on Kubernetes
 icon: https://app.rookout.com/static/rookout/img/logo.svg
+version: 0.1.1
+appVersion: 0.1.1
```
Comment (cpanato) [inline stable/rookout/Chart.yaml]: appVersion should be: 0.2.3 which is defined in your values.yaml

File: stable/rookout/README.md [hunk 5]
```diff
@@ -46,8 +46,10 @@ The following table lists the configurable parameters of the Rookout Router char
...
-| `listenAll`                       | Configuring the agent to listen on all addresses instead of only localhost.                      | `True` You must provide your own token |               
+| `listenAll`                       | Configuring the agent to listen on all addresses instead of only localhost.                      | `True` You must provide your own token |   
 | `image.registry`                  | Rookout image registry                   | `docker.io`                                             |
+| `limits.memory`                   | Memory chart limit                             | `0.5Gi`                                                 |
+| `limits.cpu`                      | CPU chart limit                                | `50m`                                                   |
 | `image.repository`                | Rookout image name                       | `rookout/agent`                                         |
 | `image.tag`                       | Rookout image tag                        | `{VERSION}`                                             |
```
Comment (cpanato) [inline stable/rookout/README.md]: instead of {VERSION} should be `0.2.3`

File: stable/rookout/templates/service.yaml [hunk 6]
```diff
@@ -15,4 +15,4 @@ spec:
   type: ClusterIP
   ports:
     - protocol: TCP
-      port: 7486
\ No newline at end of file
+      port: 7486
```
Comment (cpanato) [inline stable/rookout/templates/service.yaml]: can add a blank line in the end?

**Comment by Zohar2506** [comment]
@cpanato can it be merged?

**Comment by cpanato** [comment]
@Zohar2506 now need some chart maintainer to approve

/assign @davidkarlsen

File: stable/rookout/templates/deployment.yaml [hunk 7]
```diff
@@ -23,6 +23,10 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          resources:
+          limits:
```
Comment (unguiculus) [inline stable/rookout/templates/deployment.yaml:27]: Incorrect indentation

File: stable/rookout/values.yaml [hunk 8]
```diff
@@ -12,6 +12,10 @@ tags: ''
 rookout:
   token: ''
 
+limits:
```
Comment (unguiculus) [inline stable/rookout/values.yaml:15]: Don't set defaults for resources.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please follow this example:

https://github.com/helm/charts/blob/master/stable/mongodb-replicaset/templates/mongodb-statefulset.yaml#L151-L152

https://github.com/helm/charts/blob/master/stable/mongodb-replicaset/values.yaml#L60-L66

**Pull request closed without merging**